rules/aws_cloudtrail_rules/aws_ec2_traffic_mirroring.yml

```diff
@@ -0,0 +1,516 @@
+Severity: Medium
+Tags:
+    - AWS
+    - Cloudtrail
+    - MITRE
+Tests:
+    - ExpectedResult: true
+      Log:
+        awsRegion: us-east-1
+        eventCategory: Management
+        eventID: a3c6297b-3320-4d32-b224-cc45ee75d561
+        eventName: CreateTrafficMirrorFilter
+        eventSource: ec2.amazonaws.com
+        eventTime: "2022-11-15 22:58:13"
+        eventType: AwsApiCall
+        eventVersion: "1.08"
+        managementEvent: true
+        p_any_aws_account_ids:
+            - "123451234513"
+        p_any_aws_arns:
+            - arn:aws:iam::123451234513:role/MakeStuffPublic
+            - arn:aws:sts::123451234513:assumed-role/MakeStuffPublic
+        p_any_domain_names:
+            - AWS Internal
+        p_any_trace_ids:
+            - ASIA57JLR4M2ZZDJUXY3
+        p_any_usernames:
+            - MakeStuffPublic
+        p_event_time: "2022-11-15 22:58:13"
+        p_log_type: AWS.CloudTrail
+        p_parse_time: "2022-11-15 23:00:37.355"
+        p_row_id: 82670d2e7575bbd0e8fc97d014b5a80c
+        p_source_id: 125a8146-e3ea-454b-aed7-9e08e735b670
+        p_source_label: Panther Identity Org CloudTrail
+        readOnly: false
+        recipientAccountId: "123451234515"
+        requestID: 200a9157-dff7-4578-87d6-205b01d90a56
+        requestParameters:
+            CreateTrafficMirrorFilterRequest:
+                ClientToken: 5b7eff74-2b70-4f92-8aa1-9c716bf151aa
+        responseElements:
+            CreateTrafficMirrorFilterResponse:
+                clientToken: 5b7eff74-2b70-4f92-8aa1-9c716bf151aa
+                requestId: 200a9157-dff7-4578-87d6-205b01d90a56
+                trafficMirrorFilter:
+                    egressFilterRuleSet: ""
+                    ingressFilterRuleSet: ""
+                    networkServiceSet: ""
+                    tagSet: ""
+                    trafficMirrorFilterId: tmf-010db9a7d8056cc2d
+                xmlns: http://ec2.amazonaws.com/doc/2016-11-15/
+        sessionCredentialFromConsole: true
+        sourceIPAddress: AWS Internal
+        userAgent: AWS Internal
+        userIdentity:
+            accessKeyId: ASIA57JLR4M2ZZDJUXY3
+            accountId: "123451234516"
+            arn: arn:aws:sts::123123123123:assumed-role/MakeStuffPublic
+            principalId: AROA57JLR4M2SBAPVC4BO
+            sessionContext:
+                attributes:
+                    creationDate: "2022-11-15T22:38:17Z"
+                    mfaAuthenticated: "true"
+                sessionIssuer:
+                    accountId: "123123123123"
+                    arn: arn:aws:iam::123123123123:role/MakeStuffPublic
+                    principalId: AROA57JLR4M2SBAPVC4BO
+                    type: Role
+                    userName: MakeStuffPublic
+                webIdFederationData: {}
+            type: AssumedRole
+      Name: Test-CreateTrafficMirrorFilter
```

Comment:
this is fine, and there is no need to preface the Name with the word `Test-`.

Reply:
ah i see, i was attempted to follow the naming convention seemingly prompted by the UI, but can remove the Test- preface and update the PR